doc/files/package.json.md

```diff
@@ -733,12 +733,16 @@ The host architecture is determined by `process.arch`
 
 ## preferGlobal
 
-**DEPRECATED**
+Inform that this package is usually installed globally.
```

Comment:
Why would this no longer be deprecated? I can’t think of any reason that a *package* should decide that it’s preferred globally; that always depends on the installer’s use case.

Reply:
If we're having `preferDev` as guidance towards `devDependencies`, I though it might be weird if `preferGlobal` had the same effect but was considered deprecated. I realise installing globally isn't a meaningful distinction given `npx` and `npm run-script`, but I meant it more like a property saying "this is a CLI".

Reply:
I think preferDev makes perfect sense (and preferPeer, tbh) because the action they recommend is a best practice for that package. preferGlobal is, at best, recommending a bad practice that’s blessed by that package’s authors.